phalcon/collection.zep

```diff
@@ -108,44 +117,25 @@ class Collection implements \ArrayAccess, \Countable, \IteratorAggregate, \JsonS
 	/**
 	 * Get the element from the collection
 	 */
-	public function has(string! element) -> bool
+	public function has(string! element, bool insensitive = true) -> bool
 	{
-		return isset this->data[element];
-	}
-
-	/**
-	 * Delete the element from the collection
-	 */
-	public function remove(string! element) -> void
-	{
-		array data;
-
-		let data = this->data;
-
-		unset data[element];
-
-		this->init(data);
-	}
-
-	/**
-	 * Set an element in the collection
-	 */
-	public function set(string! element, var value) -> void
-	{
-		array data;
-
-		let data          = this->data,
-			data[element] = value;
+		if insensitive {
+			let element = strtolower(element);
+		}
 
-		this->init(data);
+		return isset this->headers[element];
 	}
 
 	/**
 	 * Initialize internal array
 	 */
-	public function init(array! data = [])
+	public function init(array! data = []) -> void
```

Comment:
How about this?
```suggestion
	internal function init(array! data = []) -> void
```

Reply:
why do we need this in the API part?

Reply:
I want to offer the option to populate the collection after it was created. For instance you can reuse the same object to populate data from another array that came in later in the code flow..